documentation/source/library_reference.rst

```diff
@@ -30,6 +30,9 @@ Writing and Generating tests
     :members:
     :member-order: bysource
 
+.. autofunction:: cocotb.decorators.RunningCoroutine.log
+
+.. autofunction:: cocotb.decorators.RunningCoroutine.join
```

Comment:
I think these would make more sense next to `fork`- you're very unlikely to get a coroutine handle unless you use that.

Reply:
Done (and added also ``kill()``, what do you think about that?)

Reply:
Good idea, `kill` definitely belongs there.

I was suggesting you move `log` as well, but I'm no longer convinced it even provides any value - see the other comment.